frontend/src/components/common/title-bar/TitleBarWithTabs.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ReactNode } from 'react'
+
+import defaultStyles from './TitleBar.module.scss'
+
+const HStackStyle = {
```

Comment:
Is there a reason the naming convention is different from the variable below? (`subsequentElementStyle`)

Reply:
`HStack` is actually a borrowed term from chakra.

I'll rename to `horizontalStackStyle`.